src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Tests.cs

```diff
@@ -266,6 +266,33 @@ public void PriorityQueue_EmptyCollection_Remove_ShouldReturnFalse()
             Assert.Null(removedPriority);
         }
 
+        [Fact]
+        public void PriorityQueue_LargeCollection_Remove_ShouldPreserveHeapInvariant()
+        {
+            // Regression test for https://github.com/dotnet/runtime/issues/107292
```

Comment:
I agree that they're not particularly useful from a testing perspective. As with every code comment they tell a story that the future code historian might appreciate (in a way that could be lost in git blame after refactoring or loss of commit history). I find that there is something noteworthy about regression tests, since they tend to encapsulate a minimal counterexample to a flawed mental model of how a piece of software should work.